src/sdk/namespace_fs.js

```diff
@@ -1582,8 +1580,11 @@ class NamespaceFS {
             });
             chunk_fs.on('error', err1 => dbg.error('namespace_fs._upload_stream: error occured on stream ChunkFS: ', err1));
             if (copy_source) {
+                // ChunkFS is a Transform stream, however read_object_stream expects a write stream. call resume to close the read part
+                chunk_fs.resume();
                 await this.read_object_stream(copy_source, object_sdk, chunk_fs);
             } else if (params.source_params) {
+                chunk_fs.resume();
```

Comment:
why don't we need to resume() on regular upload flow?
is there a chance on upload that the readable stream will change the xattr after we assigned the version id ?